app/bower.json

```diff
@@ -1,13 +1,12 @@
 {
-  "name": "NinjaTerm",
-  "version": "0.0.0",
+  "name": "beagleterm",
+  "version": "1.7.0",
```

Comment:
Interesting. Not sure how I missed that.  Thanks for point it out. I'll have to update some other projects of mine now.